presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -47,7 +47,17 @@
 #include "velox/common/memory/SharedArbitrator.h"
 #include "velox/connectors/Connector.h"
 #include "velox/connectors/hive/HiveConnector.h"
+#include "velox/connectors/hive/HiveDataSink.h"
+#include "velox/connectors/hive/storage_adapters/abfs/RegisterAbfsFileSystem.h"
+#include "velox/connectors/hive/storage_adapters/gcs/RegisterGCSFileSystem.h"
+#include "velox/connectors/hive/storage_adapters/hdfs/RegisterHdfsFileSystem.h"
+#include "velox/connectors/hive/storage_adapters/s3fs/RegisterS3FileSystem.h"
+#include "velox/connectors/tpch/TpchConnector.h"
 #include "velox/core/Config.h"
+#include "velox/dwio/dwrf/RegisterDwrfReader.h"
+#include "velox/dwio/dwrf/RegisterDwrfWriter.h"
+#include "velox/dwio/parquet/RegisterParquetReader.h"
+#include "velox/dwio/parquet/RegisterParquetWriter.h"
```

Comment:
@kgpai Do we include this in build within Meta? We need to make sure presto server compiles with Parquet (I recall its disabled inside Meta)

Reply:
@amitkdutta You are right : AbfsFileSystem, GCSFileSystem etc arent supported internally. 
@majetideepak Can we gate them with the backwards compatible flag ?

Reply:
These headers work even if optional features such as Parquet/S3/Abfs are disabled. I will remove Abfs and GCS as they are not plumbed yet to Prestissimo. There is no CMake option.

Reply:
> @amitkdutta You are right : AbfsFileSystem, GCSFileSystem etc arent supported internally. @majetideepak Can we gate them with some flag ?

@kgpai Parquet is also not compiled, and built internally. We need to gate it with macro as well if you want to include here.

Reply:
@amitkdutta the macros are present in the implementation. See Parquet for example: 
https://github.com/facebookincubator/velox/blob/main/velox/dwio/parquet/RegisterParquetReader.cpp#L24